frontend/src/app/components/ExperimentPage.js

```diff
@@ -95,6 +95,42 @@ export default class ExperimentPage extends React.Component {
     return typeof min === 'undefined' || version >= min;
   }
 
+  getIncompatibleInstalled(incompatible) {
+    if (!incompatible || !this.props.installedAddons.length) {
+      return null;
+    }
+    return Object.keys(incompatible).filter(guid => (
+      this.props.installedAddons.indexOf(guid) !== -1
+    ));
+  }
+
+  renderIncompatibleAddons() {
+    const { incompatible } = this.state.experiment;
+    if (!incompatible) return null;
+    const installed = this.getIncompatibleInstalled(incompatible);
+    if (!installed) return null;
+
+    const helpUrl = 'https://support.mozilla.org/kb/disable-or-remove-add-ons';
+
+    return (
+      <section className="incompatible-addons">
+        <header>
+          <h3 data-l10n-id="incompatibleHeader">
+            This experiment may not be compatible with Add-ons you have installed.
```

Comment:
Yeah, I didn't think so either, but I assumed @sevaan knew better than me and went with his mocks.